backend/sirius-web-spring-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -320,7 +320,7 @@ input UpdateNodePositionInput {
   id: ID!
   editingContextId: ID!
   representationId: ID!
-  diagramElementId: ID!
+  nodeId: ID!
```

Comment:
you are both modifying `UpdateNodeBoundsInput#diagramElementId` and `UpdateNodePositionInput#diagramElementId` while not related to this issue but you fail to change both of them in this file so it doesn't work anymore

Reply:
This issue was not supposed to have any impact on our GraphQL API or frontend

Reply:
Done